mergify_engine/engine/actions_runner.py

```diff
@@ -57,6 +57,44 @@
 
 """
 
+COMMIT_MESSAGE_MODE_MERGE_DEPRECATION_GHES = """
+:bangbang: **Action Required** :bangbang:
+
+> **The configuration uses the deprecated `commit_message` mode of the merge action.**
+> This option will be removed on version 3.0.0.
```

Comment:
Not sure we want to set a version number here. Between now and April we might be at version 4.0.0…

Just write "on a future version".